config/config.yaml

```diff
@@ -67,7 +67,7 @@ calling:
     # and can be referred in FDR control below to generate calls for different events.
     # In particular, you can also filter by ID or dbsnp annotations here.
     # See http://snpeff.sourceforge.net/SnpSift.html#filter
-    filtername: "ANN[*].IMPACT has 'MODERATE'"
+    filtername: \"ANN['IMPACT'] == 'MODERATE'\"
```

Comment:
Please move the quotation into the rule (snakemake has `{params.filter:q}` which will take care of quoting and escapes. No need to bug people in here with escaping quotes.